src/util.ts

```diff
@@ -133,3 +137,29 @@ export const getNextAgentOffset = (
     const nSubPoints = view[currentOffset + AGENT_OBJECT_KEYS.length - 1];
     return currentOffset + AGENT_OBJECT_KEYS.length + nSubPoints;
 };
+
```

Comment:
I know this is still a draft, but I would want to see how you're planning on using these